Pipfile

```diff
@@ -4,15 +4,14 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pylint = "*"
-pytest = "*"
-pytest-mock = "*"
-pre-commit = "*"
-
-[packages]
-aws-sam-cli = "*"
-boto3 = "*"
-botocore = "*"
+aws-sam-cli = "~=1.2.0"
+boto3 = "~=1.14.58"
+botocore = "~=1.17.58"
+jsondiff = "~=1.2.0"
+pre-commit = "~=2.7.1"
+pylint = "~=2.6.0"
```

Comment:
don't need this because we are using pre-commit

Reply:
fixed

Reply:
pylint is not needed, please remove.